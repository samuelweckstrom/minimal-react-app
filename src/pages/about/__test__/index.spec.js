import React from 'react'
import { shallow } from 'enzyme'
import About from '../'

describe('About', () => {
  const wrapper = shallow(<About />)

  it('It matches snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
