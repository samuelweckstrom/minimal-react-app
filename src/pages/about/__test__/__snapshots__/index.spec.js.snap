// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About It matches snapshot 1`] = `ShallowWrapper {}`;
